src/precompile.js

```diff
@@ -0,0 +1,14 @@
+const jade = require('jade');
+const assign = require('lodash.assign');
+
+const defaults = {
+  compileDebug: false,
+  name: 't'
+};
+
+module.exports = (template, options) => {
+  const opts = assign({}, defaults, options);
```

Comment:
The only thing I don't like about this is that we pass the view path as "filename" because jade very badly named this. That's why I purposed on my previous snippet to have something here like `filename: options.viewPath` which sounds more nice for a generic interface I think. Consider that in pug this is the same (I just checked) - so I guess on the other side (oc) we would need to do something like `filename: viewPath` where I would just prefer to send `{ viewPath }` on that side